tests/sqllogictests/suites/base/05_ddl/05_0000_ddl_drop_tables

```diff
@@ -13,6 +13,6 @@ DROP TABLE IF EXISTS t
 statement error 1025
 DROP TABLE t
 
-statement error 1002
+statement error 1025
 DROP TABLE system.abc
```

Comment:
It would be better to add:
DROP TABLE IF EXISTS system.abc

Reply:
Add test for 

drop table/view; 

alter table xx drop column